app/services/course_search_service.rb

```diff
@@ -11,13 +11,12 @@ def initialize(
     @sort = Set.new(sort&.split(","))
   end
 
-  PROVIDER_ASCENDING = Set["name", "provider.provider_name"].freeze
```

Comment:
I think this set which fields to order on (eg course name and the provider name in this case). Without it, it's just ordering on the provider name:

Review app:

<img width="796" alt="Screenshot 2022-12-02 at 12 16 19" src="https://user-images.githubusercontent.com/616080/205291012-34f3ef4f-ad84-4cee-a19d-1d17428a280c.png">

Find:

<img width="772" alt="Screenshot 2022-12-02 at 12 16 36" src="https://user-images.githubusercontent.com/616080/205291078-42842ffe-311a-471f-a603-44b91ce6e36e.png">

Reply:
cc @defong

Reply:
This look like a bad merge...

Reply:
I think the sorting is done by `ascending_canonical_order`? It seems to be working ok now